proposals/0002-web-rtc.md

```
@@ -0,0 +1,329 @@
+| Accepted    | No                                                                                      |
+| Issue       | [KTOR-7958 WebRTC Client](https://youtrack.jetbrains.com/issue/KTOR-7958/WebRTC-Client) |
+| Preceded by |                                                                                         |
+| Followed by |                                                                                         |
+
+### Contents
+
+1. [Summary](#summary)
+2. [Motivation](#motivation)
+3. [Current Solutions](#current-solutions)
+4. [Design Overview](#design-overview)
+5. [Background: WebRTC, ICE, SDP, RTP](#background)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)